tests/IResearch/common.cpp

```diff
@@ -410,13 +410,13 @@ std::unique_ptr<arangodb::aql::ExecutionPlan> planFromQuery(
   return arangodb::aql::ExecutionPlan::instantiateFromAst(query.ast());
 }
 
-std::unique_ptr<arangodb::aql::ExecutionPlan> optimizedPlanFromQuery(
+std::unique_ptr<arangodb::aql::Query> prepareQuery(
   TRI_vocbase_t& vocbase,
   std::string const& queryString,
   std::shared_ptr<arangodb::velocypack::Builder> bindVars /* = nullptr */,
   std::string const& optionsString /*= "{}"*/
 ) {
-  arangodb::aql::Query query(
+  auto query = std::make_unique<arangodb::aql::Query>(
     false,
     vocbase,
     arangodb::aql::QueryString(queryString),
```

Comment:
I was just wondering this is correct. https://stackoverflow.com/questions/2784262/does-a-const-reference-class-member-prolong-the-life-of-a-temporary